src/main/java/mops/businesslogic/Account.java

```diff
@@ -19,13 +22,39 @@
      */
     @NonNull
     private final String email;
+    /**
+     * Keycloak roles of the user.
+     */
+    @NonNull
+    private final Set<String> roles;
     /**
      * Avatar of the user.
      */
     private String image;
+
     /**
-     * Keycloak roles of the user.
+     * Create a new Account.
+     *
+     * @param name  user name
+     * @param email email address
+     * @param roles permission roles
      */
-    @NonNull
-    private final Set<String> roles;
+    public Account(String name, String email, String... roles) {
+        this.name = Objects.requireNonNull(name);
+        this.email = Objects.requireNonNull(email);
+        this.roles = Set.of(roles);
+    }
```

Comment:
I don't think we need this constructor, because we will always create a account from token where it is already a set. And then it is easier with lombok annotation - less boiler plate code.

Reply:
In the tests we don't. If we do this we save a lot of code and the line length will stay under 120 without ugly linebreaks